ReplayKitExample/BroadcastExtension/SampleHandler.swift

```diff
@@ -110,14 +110,12 @@ class SampleHandler: RPBroadcastSampleHandler, TVIRoomDelegate {
         case RPSampleBufferType.video:
             videoSource?.processVideoSampleBuffer(sampleBuffer)
             break
+
         case RPSampleBufferType.audioApp:
-            /*
-             * TODO: We do not capture app audio at the moment. For some broadcast use cases it may make sense to capture both the
-             * application and microphone audio. Doing this requires down-mixing the resulting streams.
-             */
+            ExampleCoreAudioDeviceRecordCallback(sampleBuffer)
             break
+
         case RPSampleBufferType.audioMic:
-            ExampleCoreAudioDeviceRecordCallback(sampleBuffer)
```

Comment:
So no more microphone audio? Just app audio?

Reply:
I don't understand why we would take away the existing feature of capturing from the microphone audio. There should be an option to do either, and it should probably default to the microphone.

Reply:
It was a mistake. Adding it back.